tests/unit/qunit/async-iterator-test.js

```diff
@@ -6,7 +6,7 @@ import {
 } from '@embroider/macros';
 
 if (macroCondition(dependencySatisfies('ember-qunit', '*'))) {
-  let { module, test } = importSync('qunit');
+  let { module, test } = importSync('qunit').default;
```

Comment:
I assume previous versions of `@embroider/macro` happily "swallowed" qunit not being an ES module but now we need to be explicit.

I assume this has changed in https://github.com/embroider-build/embroider/pull/1076 ad should be way to move forward in such scenarios.

@ef4 would appreciate you can confirm

Reply:
I think you're correct. It was a bug that we allowed this to work before with `.default`.